BackofficeBundle/Resources/config/navigation_panel.yml

```diff
@@ -123,14 +123,14 @@ services:
             - 334
         tags:
             - { name: open_orchestra_backoffice.navigation_panel.strategy }
-    open_orchestra_backoffice.navigation_panel.template_flex:
-        class: %open_orchestra_backoffice.navigation_panel.template_flex.class%
-        arguments:
-            - @open_orchestra_model.repository.template_flex
-            - %open_orchestra_backoffice.navigation_panel.menu.administration%
-            - 336
-        tags:
-            - { name: open_orchestra_backoffice.navigation_panel.strategy }
+#    open_orchestra_backoffice.navigation_panel.template_flex:
+#        class: %open_orchestra_backoffice.navigation_panel.template_flex.class%
+#        arguments:
+#            - @open_orchestra_model.repository.template_flex
+#            - %open_orchestra_backoffice.navigation_panel.menu.administration%
+#            - 336
+#        tags:
+#            - { name: open_orchestra_backoffice.navigation_panel.strategy }
```

Comment:
I 'don't know if it's the better way of doing this. I'd put that conf in a specific yaml not loaded to avoid comment in operational file. What do you think about this ?